src/validators/datetime.rs

```diff
@@ -199,3 +209,37 @@ impl NowConstraint {
         }
     }
 }
+
+#[derive(Debug, Clone)]
+pub enum TZConstraintKind {
+    Aware,
+    Naive,
+}
+
+impl TZConstraintKind {
+    pub fn from_str(s: &str) -> PyResult<Self> {
+        match s {
+            "aware" => Ok(TZConstraintKind::Aware),
+            "naive" => Ok(TZConstraintKind::Naive),
+            _ => py_err!("Invalid tz_constraint {:?}", s),
+        }
+    }
+}
+
+#[derive(Debug, Clone)]
+pub struct TZConstraint {
```

Comment:
do we need this, can't we just use `TZConstraintKind`?

(and I guess you could rename `TZConstraintKind` -> `TZConstraint`)